app/views/subscriptions/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= simple_form_for subscription do |f| %>
+  <%= f.input :email %>
```

Comment:
<img width="700" alt="screen shot 2016-04-01 at 9 15 25 am" src="https://cloud.githubusercontent.com/assets/2575027/14212974/501b91ac-f7ea-11e5-94b2-844da3d98943.png">

@croaky this was news to me. Nifty!